templates/default/papertrail.conf.erb

```diff
@@ -20,12 +20,9 @@ $ActionQueueSaveOnShutdown on
 $ActionQueueTimeoutEnqueue 10
 $ActionQueueDiscardSeverity 0
 
-<%- if @fixhostname %>
-
-*.*   @@<%= @host %>:<%= @port %>;FixHostname
-
-<%- else %>
-
-*.*   @@<%= @host %>:<%= @port %>
-
-<%- end %>
+<%   case node['papertrail']['protocol'] -%>
```

Comment:
The TCP vs. UDP change is at least a little bigger than this, in that none of the Action* directives affect or should be generated for UDP. I'd recommend splitting the protocol support and config file change into a separate PR, since it's not related to wildcard filenames.